src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: string;
 
-}
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.shape = 'triangle';
+
+    function properTriangle(
+      aSide: number,
+      bSide: number,
+      cSide: number,
+    ): boolean {
+      const allSides = [aSide, bSide, cSide];
```

Comment:
you can also sort them right away, and the max side would be either the first element of the array or the last one, depending on the asc/desc sorting